pkg/shootflavors/extendedflavors_test.go

```diff
@@ -439,6 +439,44 @@ var _ = Describe("extended flavor test", func() {
 		Expect(shoot.Get().ExtendedConfiguration).To(Equal(defaultExtendedCfg))
 	})
 
+	It("should generate a shoot with the correct ipFamilies set", func() {
+		defaultExtendedCfg.IpFamilies = "ipv4"
```

Comment:
Gardener uses `IPv4` and `IPv6` (see https://github.com/gardener/gardener/blob/ab89d09e2d6b2cb6d34b46bcc2186a94020b6a9e/pkg/apis/core/v1beta1/types.go#L18-L25).